x-pack/plugins/security_solution/public/cases/components/all_cases/columns.tsx

```diff
@@ -121,9 +130,21 @@ export const getCasesColumns = (
       },
       truncateText: true,
     },
+    {
+      align: 'right' as HorizontalAlignment,
```

Comment:
```suggestion
      align: RIGHT_ALIGNMENT
```

and `import { RIGHT_ALIGNMENT } from '@elastic/eui/lib/services';`